backend/business-partner-agent/pom.xml

```diff
@@ -120,7 +120,7 @@
         <dependency>
             <groupId>org.hyperledger</groupId>
             <artifactId>aries-client-python</artifactId>
-            <version>0.14.0</version>
+            <version>0.20.0</version>
```

Comment:
if you merge the main branch you will get version 0.21.0 which contains the handler for the problem report.